zoo/policies/open-agent/open_agent/agent.py

```diff
@@ -504,8 +504,8 @@ def act(self, obs):
             + flat_wps
             + [impatience, wps[0].speed_limit]
         )
-
-        resp = self.solver.run(solver_params, initial_guess=self.prev_solution)
+        aaa = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
nit: Perhaps there is a more descriptive variable name here: `neutral_bias`?

Reply:
`[0 for i in range(12)]` ?

Reply:
Thanks, I also checked and it is working fine with uturn,